src/CMakeLists.txt

```diff
@@ -142,11 +142,18 @@ function(ov_tokenizers_link_sentencepiece TARGET_NAME)
 endfunction()
 
 if(BUILD_FAST_TOKENIZERS)
+  set(EXTERNAL_PROJECT_SOURCE_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/src)
+  set(EXTERNAL_PROJECT_BINARY_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/build)
+  set(EXTERNAL_PROJECT_SUBBUILD_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/sub-build)
```

Comment:
why do we need non-default directories?

Reply:
We need to know the root source path of the external project to apply properly the patch (using git apply)